GameData/002_AdditionalConfigs/Knes.cfg

```diff
@@ -717,20 +854,46 @@
     %entryCost = 75000
     RP0conf = true
     @description ^=:$: <b><color=green>From Knes mod</color></b>
+
     %MODULE[ModuleTagList] { tag = HumanRated }
     %MODULE[ModuleTagList] { tag = NoResourceCostMult }
     %MODULE[ModuleTagList] { tag = Reentry }
-	%identicalParts = Knes_Crew_Pod_09375,Knes_Service_Module_Viking_09375
+
+	%identicalParts = _Knes_Control_LowProfile_RCS_0625,Knes_DockingPortJr_0625,Knes_Parachute_Inline_0625,Knes_Service_Module_Viking_09375
+
 }
+@PART[Knes_EscapeTower_Small]:FOR[xxxRPO]
+{
+	%TechRequired = basicCapsules
+    %cost = 250
+    %entryCost = 25000
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Knes mod</color></b>
+
+	%MODULE[ModuleTagList] { tag = NoResourceCostMult }
+}
+
+@PART[_Knes_Control_LowProfile_RCS_0625,Knes_DockingPortJr_0625,Knes_Parachute_Inline_0625]:FOR[xxxRPO]
+{
+	%TechRequired = basicCapsules
+    %cost = 50
+    %entryCost = 75000
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Knes mod</color></b>
+	%MODULE[ModuleTagList] { tag = NoResourceCostMult }
+}
+
 @PART[Knes_Service_Module_Viking_09375]:FOR[xxxRP0]
 {
     %TechRequired = basicCapsules
     %cost = 650
     %entryCost = 75000
     RP0conf = true
     @description ^=:$: <b><color=green>From Knes mod</color></b>
+
     %MODULE[ModuleTagList] { tag = NoResourceCostMult }
-	%identicalParts = Knes_Crew_Pod_09375,Knes_Service_Module_Viking_09375
+	%identicalParts = _Knes_Control_LowProfile_RCS_0625,Knes_DockingPortJr_0625,Knes_Parachute_Inline_0625,Knes_Crew_Pod_09375
```

Comment:
This was only a workaround when I didn't know how entry cost modifiers worked. Now that the parts are properly costed, all of these identicalParts declarations can just be deleted. It should only be used for actual identical parts, for example Gemini capsules added by two different mods. There is no need for its usage in Knes